File: src/libs/ThirdParty.VBSuppliers/Properties/Travelgate/Thirdparties/TravelgateDingus/TravelgateDingusSearch.vb

Imports ThirdParty.Abstractions.Constants
Imports ThirdParty.Abstractions.Lookups

Public Class TravelgateDingusSearch
    Inherits TravelgateSearch

    Public Sub New(settings As ITravelgateSettings, support As ITPSupport)
        MyBase.New(settings, support)
    End Sub

    Public Overrides Property Source As String
        Get
            Return ThirdParties.TRAVELGATEDINGUS
        End Get
        Set
        End Set
    End Property

End Class

File: src/libs/ThirdParty.CSSuppliers/Travelgate/Thirdparties/TravelgateDingus/TravelgateDingusSearch.cs

using Intuitive.Helpers.Security;
using Intuitive.Helpers.Serialization;
using ThirdParty.Constants;
using ThirdParty.Lookups;

namespace ThirdParty.CSSuppliers
{

    public class TravelgateDingusSearch : TravelgateSearch
    {

        public TravelgateDingusSearch(ITravelgateSettings settings, ITPSupport support, ISecretKeeper secretKeeper, ISerializer serializer) : base(settings, support, secretKeeper, serializer)
        {
        }

        public override string Source
        {
            get
            {
                return ThirdParties.TRAVELGATEDINGUS;
            }
        }

    }
}